picamera2/picamera2.py

```diff
@@ -210,9 +210,8 @@ def __init__(self, camera_num=0, verbose_console=None, tuning=None):
         :type tuning: str, optional
         :raises RuntimeError: Init didn't complete
         """
-        self.log = _logger
         if verbose_console is not None:
-            self.log.warn("verbose_console parameter is no longer used, use Picamera2.set_logging instead")
```

Comment:
Also note that `.warn()` is deprecated and doesn't appear in the docs since 3.8 or so.